yb-voyager/src/dbzm/config.go

```diff
@@ -251,6 +278,9 @@ debezium.source.database.sslrootcert=%s
 func (c *Config) String() string {
 	dataDir := filepath.Join(c.ExportDir, "data")
 	offsetFile := filepath.Join(dataDir, "offsets.dat")
+	if isTargetDBExporter(c.ExporterRole) {
```

Comment:
Instead of checking, just set the offsetFile path to "offsets.<exportRole>.data". That will automatically take care of separating it out.